backtester/report/tpl.gohtml

```diff
@@ -0,0 +1,710 @@
+									<td>{{ printf "%.2f" $val.StrategyMovement}}%</td>
+								</tr>
+								<tr>
+									<td><b>Did it beat the market?</b></td>
+									<td>{{ gt .StrategyMovement $val.MarketMovement}}</td>
+								</tr>
+								<tr>
+									<td><b>Total Value Lost to Volume Sizing</b></td>
+									<td>${{printf "%.8f" $val.FinalHoldings.TotalValueLostToVolumeSizing}} {{$val.FinalHoldings.Pair.Quote}}</td>
+								</tr>
+								<tr>
+									<td><b>Total Value Lost to Slippage</b></td>
+									<td>${{printf "%.8f" $val.FinalHoldings.TotalValueLostToSlippage}} {{ $val.FinalHoldings.Pair.Quote }}</td>
+								</tr>
+								<tr>
+									<td><b>Total Value Lost</b></td>
+									<td>${{printf "%.8f" $val.FinalHoldings.TotalValueLost}} {{$val.FinalHoldings.Pair.Quote}}</td>
+								</tr>
+								<tr>
+									<td><b>Total Fees</b></td>
+									<td>${{printf "%.2f" $val.FinalHoldings.TotalFees}} {{ $val.FinalHoldings.Pair.Quote }}</td>
+								</tr>
+								<tr>
+									<td><b>Final Funds</b></td>
+									<td>${{printf "%.2f" $val.FinalHoldings.RemainingFunds}} {{ $val.FinalHoldings.Pair.Quote}}</td>
+								</tr>
+								<tr>
+									<td><b>Final Holdings</b></td>
+									<td>{{printf "%.8f" $val.FinalHoldings.PositionsSize}} {{$val.FinalHoldings.Pair.Base}}</td>
+								</tr>
+								<tr>
+									<td><b>Final Holdings Value</b></td>
+									<td>${{printf "%.8f" $val.FinalHoldings.PositionsValue}} {{ $val.FinalHoldings.Pair.Quote }}</td>
+								</tr>
+								<tr>
+									<td><b>Total Value</b></td>
+									<td><b>${{printf "%.2f" $val.FinalHoldings.TotalValue}} {{ $val.FinalHoldings.Pair.Quote}}</b></td>
+								</tr>
+								</tbody>
+							</table>
+							Rates
+							<table class="table table-hover table-bordered table-striped">
+								<tbody>
+								<tr>
+									<td><b>Risk Free Rate</b></td>
+									<td>{{$val.RiskFreeRate}}%</td>
+								</tr>
+								<tr>
+									<td><b>Compound Annual Growth Rate</b></td>
+									<td>{{$val.CompoundAnnualGrowthRate}}%</td>
+								</tr>
+								</tbody>
+							</table>
+							Arithmetic Ratios
+							<table class="table table-hover table-bordered table-striped">
+								<tbody>
+								<tr>
+									<td><b>Sharpe Ratio</b></td>
+									<td>{{$val.ArithmeticRatios.SharpeRatio}}</td>
+								</tr>
+								<tr>
+									<td><b>Sortino Ratio</b></td>
+									<td>{{$val.ArithmeticRatios.SortinoRatio}}</td>
+								</tr>
+								<tr>
+									<td><b>Information Ratio</b></td>
+									<td>{{$val.ArithmeticRatios.InformationRatio}}</td>
+								</tr>
+								<tr>
+									<td><b>Calmar Ratio</b></td>
+									<td>{{$val.ArithmeticRatios.CalmarRatio}}</td>
+								</tr>
+								</tbody>
+							</table>
+							Geometric Ratios
+							<table class="table table-hover table-bordered table-striped">
+								<tbody>
+								<tr>
+									<td><b>Geometric Sharpe Ratio</b></td>
+									<td>{{$val.GeometricRatios.SharpeRatio}}</td>
```

Comment:
Since the title is Geometric Ratios I think the subtitle should just be Sharpe Ratio rather than Geometric Sharpe Ratio